rocAL/rocAL_pybind/rocal_pybind.cpp

```diff
@@ -666,111 +482,119 @@ namespace rocal{
+        // m.def("Caffe_ImageDecoder", &rocalJpegCaffeLMDBRecordSource,"Reads file from the source given and decodes it according to the policy only for TFRecords",
+        //     py::return_value_policy::reference);
+        // m.def("Caffe_ImageDecoderShard", &rocalJpegCaffeLMDBRecordSourceSingleShard, "Reads file from the source given and decodes it according to the shard id and number of shards",
+        //     py::return_value_policy::reference);
+        // m.def("Caffe_ImageDecoderPartialShard", &rocalJpegCaffeLMDBRecordSourcePartialSingleShard);
+        // m.def("Caffe2_ImageDecoder", &rocalJpegCaffe2LMDBRecordSource,"Reads file from the source given and decodes it according to the policy only for TFRecords",
+        //     py::return_value_policy::reference);
+        // m.def("Caffe2_ImageDecoderShard", &rocalJpegCaffe2LMDBRecordSourceSingleShard,"Reads file from the source given and decodes it according to the shard id and number of shards",
+        //     py::return_value_policy::reference);
+        // m.def("Caffe2_ImageDecoderPartialShard", &rocalJpegCaffe2LMDBRecordSourcePartialSingleShard);
+        // m.def("FusedDecoderCrop", &rocalFusedJpegCrop,"Reads file from the source and decodes them partially to output random crops",
+        //     py::return_value_policy::reference);
+        // m.def("FusedDecoderCropShard", &rocalFusedJpegCropSingleShard,"Reads file from the source and decodes them partially to output random crops",
+        //     py::return_value_policy::reference);
+        // m.def("TF_ImageDecoderRaw", &rocalRawTFRecordSource,"Reads file from the source given and decodes it according to the policy only for TFRecords",
+        //       py::return_value_policy::reference);
+        // m.def("Cifar10Decoder", &rocalRawCIFAR10Source,"Reads file from the source given and decodes it according to the policy only for TFRecords",
+        //       py::return_value_policy::reference);
+        m.def("videoDecoder", &rocalVideoFileSource, "Reads videos from the source given and decodes it according to the policy only for videos as inputs",
+            py::return_value_policy::reference);
+        m.def("videoDecoderResize", &rocalVideoFileResize, "Reads videos from the source given and decodes it according to the policy only for videos as inputs. Resizes the decoded frames to the dest width and height.",
+            py::return_value_policy::reference);
+        m.def("sequenceReader", &rocalSequenceReader, "Creates JPEG image reader and decoder. Reads [Frames] sequences from a directory representing a collection of streams.",
+            py::return_value_policy::reference);
+        m.def("rocalResetLoaders", &rocalResetLoaders);
+        m.def("videoMetaDataReader", &rocalCreateVideoLabelReader, py::return_value_policy::reference);
         // rocal_api_augmentation.h
-        m.def("SSDRandomCrop",&rocalSSDRandomCrop,
+        // m.def("SSDRandomCrop", &rocalSSDRandomCrop,
```

Comment:
@fiona-gladwin why is this commented out? I see SSD Random crop was added in fn.py

Reply:
Uncommented the SSD random crop augmentation in pybind